docs/config.adoc

```diff
@@ -2307,6 +2308,59 @@ when activated by the override which is important to consider for your use cases
 )
 ----
 
+[[allow-hardware-repeat]]
+=== allow-hardware-repeat
+<<table-of-contents,Back to ToC>>
+
+By default, any repeat-key events generated by the physical keyboard (or operating system)
+will be passed through to the application.  On Linux, this is known to cause some problems;
+see https://github.com/jtroo/kanata/discussions/422[discussion #422] and https://github.com/jtroo/kanata/issues/450[issue #450].
+
+Setting this option to "false" will cause such events to be dropped, and not passed through.
```

Comment:
Does this fix the problems? I would expect not, but happy to be proven wrong.

What I remember of this problem is that the fault does not lie with the kernel-level repeat events. The problem is that the desktop system **ignores** kernel-level repeat events and instead relies on its own timings purely from reading down/up events. That was my knowledge of the issue on Linux though; perhaps the situation is different on macOS? In which case if it fixes macOS and not Linux, this should be a macOS-only configuration.

Reply:
You're right.  I misinterpreted your explanation of the problem from the initial problem report.  The patch was actually inspired for other reasons, and I had just been happy thinking it solved the #422 issue as well.   Perhaps it's not something you'll feel is worth merging still, and if not, i'm happy to drop it from the set.